File: Java/decimoEjercicio.java
import java.util.Random;
import java.util.Scanner;

public class decimoEjercicio {
    public static void main(String[] args) {
        int intentos = 10;
        Scanner scanner = new Scanner(System.in);
        Random random = new Random();
        System.out.println("Simulador de lanzamientos de dados");
        System.out.println("Tienes 10 intentos");
        System.out.println("1. lanzar los dados.\n 2. Dejarlos quietos");
        System.out.println("Digita una opcion");
        int op = esEntero(scanner);
        switch (op) {
            case 1:
                for (int i = 0; i < intentos; i++) {
                    int elemento1 = random.nextInt(1,6);
                    int elemento2 =random.nextInt(1,6);
                    int[]vector1={elemento1};//Aqui se van a almacenar las jugadas del dado1
                    int[]vector2 ={elemento2}; //Aqui lo mismo pero del dado 2

                    System.out.println("Dado 1: " + elemento1 + "| Dado 2: " + elemento2);
                    if (elemento1==elemento2){
                        System.out.println("Has sacado par ¡Ganado!");
                    }
                    System.out.println(" ");

                }

                break;
            case 2:
                System.out.println("Has dejado quieto los dados");

            default:
                System.out.println("Opción inválida ¡Adiós! \n By: Daniel Sánchez" );
                break;
        }

    }

    public static int esEntero(Scanner scanner) {
        while (true) {
            try {
                return Integer.parseInt(scanner.next());
            } catch (NumberFormatException e) {
                System.out.println("Error: ingrese un nuevo entero");

            }

        }
    }
}

File: Python/decimo_ejercicio.py
from utils import es_numero_entero
import random
from rich.console import Console
console = Console()
def dados(intentos=10):
    console.print("[bold magenta]Simulador de lanzamientos de dados[bold magenta]")
    console.print("[bold yellow]Tienes 10 intentos[bold yellow]")
    console.print("[bold green]1. lanzar los dados.\n 2. Dejarlos quietos[bold green]")
    op=es_numero_entero(console.input("[italic blue]Digita un numero\n[italic blue]"))
    if op==1:
        for i in range(intentos):
            dado_uno=random.randint(1,6)
            dado_dos=random.randint(1,6)
            vector_uno=[dado_uno]#Aqui se van a almacenar las jugadas del dado1
            vector_dos=[dado_dos]#Aqui lo mismo pero del dado 2
            console.print(f"[italic blue]Dado 1 : {dado_uno} || Dado 2 : {dado_dos}[italic blue]")
            if dado_uno==dado_dos:
                console.print(f"[italic green]Has sacado par ¡Ganado![italic green]")
            console.print(" ")
        console.print("[italic red]By: Daniel Sánchez[italic red]")
    elif op ==2:
        console.print(f"[code red]Has dejado quieto los dados[code red]")
    else:
        console.print(f"[bold red]Opción inválida ¡Adiós![bold red]")

if __name__=="__main__":
    dados()
